RedHatAI/DeepSeek-R1-Distill-Llama-8B-FP8-dynamic/accuracy/client.yml

```diff
@@ -0,0 +1,8 @@
+# llm-eval-test configs for https://huggingface.co/RedHatAI/DeepSeek-R1-Distill-Llama-8B-FP8-dynamic
+model: "vllm"
+model_args:
+  pretrained: "RedHatAI/DeepSeek-R1-Distill-Llama-8B-FP8-dynamic"
+num_fewshot:
```

Comment:
Should this field be set?

Reply:
For now I'll set it to 0 and allow people to overwrite it if needed.